tb_plugin/torch_tb_profiler/profiler/data.py

```diff
@@ -66,7 +66,22 @@ def parse(run_dir, worker):
             # Kineto may export json file with non-ascii code. before this is fixed, use a workaround
```

Comment:
is N/A case still `json.decoder.JSONDecodeError`?

Reply:
yes